### PR TITLE
Portfolio: header/headline section

### PR DESCRIPTION
**Pull request by virag-ky**
I added the mobile version of the navbar and the headline section with a vector design.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,57 @@
...
-  <body></body>
+  <body>
+    <!--Header with navbar-->
+    <header>
+      <nav>
+        <ul>
+          <li><span>Virag Kormoczy</span></li>
+          <li>
+            <a href="#"><i class="fas fa-bars"></i></a>
+          </li>
```
Comment (thecodechaser) [inline index.html:34]: - [ ] Your hamburger icon is not showing on the browser, Please fix the issue so it appears correctly on the browser.

Please check this image for reference

Screenshot (45) (1)

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @virag-ky👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a title and summary. :heavy_check_mark:
- Code is clean. :heavy_check_mark:
- Good commit messages. :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -4,7 +4,61 @@
...
-  <body></body>
+  <body>
+    <!--Header with navbar-->
+    <header>
+      <nav>
+        <ul>
+          <li><span>Virag Kormoczy</span></li>
+          <li>
+            <a href="#"
+              ><img src="menu-4-24.png" alt="hamburger menu" id="menu"
```
Comment (mateo951) [inline index.html:30]: Addressed 👌

Captura1

**Review by mateo951 — APPROVED** [review]
### Hi @virag-ky,
 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🙌

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**